h2o-algos/src/main/java/hex/ensemble/StackedEnsembleModel.java

```diff
@@ -205,10 +220,16 @@ boolean isUsefulBaseModel(Key<Model> baseModelKey) {
   }
 
   ModelMetrics doScoreMetricsOneFrame(Frame frame, Job job) {
-      Frame pred = this.predictScoreImpl(frame, new Frame(frame), null, job, true, CFuncRef.from(_parms._custom_metric_func));
-//      Frame pred = this.score(frame, null, job, true,  CFuncRef.from(_parms._custom_metric_func));
+    Frame scoredFrame = (_parms._score_training_samples > 0 && _parms._score_training_samples < frame.numRows())
```

Comment:
can we apply this logic in `doScoreOrCopyMetrics`, that is where we know for sure that the frame is the training frame?

Reply:
or rename this method to ensure it can't be used for anything else than training frame.. even wondering why this is not private.

Reply:
please make it private and rename to `doScoreTrainingMetrics` or sth similar.

Reply:
Sure, good point. Thanks!